packages/clients/dish/src/plugins/DishExportMap/DishExportMap.vue

```diff
@@ -130,6 +131,7 @@ export default Vue.extend({
       this.exportMapAsPdfUrl ||= `${this.internalHost}/Content/Objekt/Kartenausgabe.aspx`
     },
     showRectangleAndDialog() {
+      if (this.isInFullscreen) document.exitFullscreen()
```

Comment:
```suggestion
      if (this.isInFullscreen) {
        if (document.exitFullscreen) {
          document.exitFullscreen()
          // @ts-expect-error | Error: 'TS2339: Property 'webkitExitFullscreen' does not exist on type 'Element'.'; For information refer to https://developer.mozilla.org/en-US/docs/Web/API/Fullscreen_API#browser_compatibility
        } else if (document.webkitExitFullscreen) {
          // @ts-expect-error | Error: 'TS2339: Property 'webkitExitFullscreen' does not exist on type 'Element'.'; For information refer to https://developer.mozilla.org/en-US/docs/Web/API/Fullscreen_API#browser_compatibility
          document.webkitExitFullscreen() // iOS Safari
        }
        this.setIsInFullscreen(!this.isInFullscreen)
      }
```

I tried to fix this, but the reference to `this.$root.$el` was not easy to fetch from the store. In getters, we don't have a `this`, and then I tried to write a hacky action that requires the calling components to provide their `this` context from which I could have retrieved the root element ... at that point I figured code repetition is probably the better solution.